src/core/Akka.Streams/Stage/GraphStage.cs

```diff
@@ -423,6 +425,16 @@ public Timer(int id, ICancelable task)
     /// </summary>
     public abstract class GraphStageLogic : IStageLogging
     {
+        private List<Func<object, Task>>? CallbacksWaitingForInterpreter = null;
```

Comment:
Still need to implement this.

Technically, callbacks can be invoked early before the interpreter is initialized. This should cache these early callbacks and defer their invocation until the interpreter is initialized.

Reply:
Implemented.